src/main/java/uk/gov/justice/hmpps/datacompliance/events/publishers/dto/DataDuplicateCheckEvent.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.hmpps.datacompliance.events.publishers.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class DataDuplicateCheckEvent {
+
+    @JsonProperty("offenderIdDisplay")
```

Comment:
as a matter of interest do you need these annotations or do you think it in good practice to keep the names independent to the property names?

Reply:
I've always added this annotation I think mainly to remind developers looking at the code that the serialised name matters. I can see that there are negatives to this too though, for instance it could lead to a refactored field name becoming out of step with how it's serialised